### DISCUSSION TITLE
Where is the flaresolverr.service example?

### DISCUSSION THREAD

**Discussion by retiolus** [FlareSolverr/FlareSolverr#996] [General]
Where is the flaresolverr.service example?

**Comment by Darlio** [comment]
Hi there I was looking for it as well... it is a Systemd service here : 
/etc/systemd/system/flaresolverr.service

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr/commit/805a34c9d621cf39c04ca799a5df26ac85dcfd56

**Reply by retiolus** [reply]
thank you
